packages/gatsby/src/utils/webpack.config.js

```diff
@@ -230,6 +230,7 @@ module.exports = async (
                         `/` +
                         files.filter(
                           f =>
+                            f.slice(-4) !== `.css` &&
```

Comment:
should this target `js` files specifically instead of blacklisting other types? i.e. would it be possible that images would be in this list if someone use `import someImage from "./assets/logo.png"`? and if it would should this be prefetched?

Reply:
Hmm yeah, that'd make more sense and be more readable. These are the webpack chunk files though and the only chunk types are for js and now css w/ webpack 4. Images are turned into modules. Chunks are groups of modules.